src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgeryTokenGenerator.cs

```diff
@@ -66,7 +66,12 @@ public AntiforgeryToken GenerateCookieToken()
             if (identity != null && identity.IsAuthenticated)
```

Comment:
Should we still check this now that you're checking `User?.Identities`? Seems rendundant